--- mesh_loader.cpp.orig	2016-04-04 08:06:44.000000000 +0800
+++ mesh_loader.cpp	2016-08-07 15:52:51.064382748 +0800
@@ -50,7 +50,6 @@
 
 #include <tinyxml.h>
 
-
 #include <ros/assert.h>
 
 #if defined(ASSIMP_UNIFIED_HEADER_NAMES)
@@ -69,6 +68,13 @@
 
 namespace fs = boost::filesystem;
 
+#ifdef __arm__
+bool Assimp::IOSystem::ComparePaths(const char *p1, const char *p2) const
+{
+   return !::strcasecmp(p1,p2);
+}
+#endif
+
 namespace rviz
 {
 
